src/components/Errors.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  showError: boolean,
+  setShowError: (value: boolean) => void,
+  errorMessage: string,
```

Comment:
I don't think that you need both `showError` and `errorMessage` . 
If there is an `errorMessge` then show it. 
if you want to hide the error, just  clear the `errorMessage`

Reply:
The thing is, if I set the errorMessage to an empty string, the error message disappears abruptly, and I believe it should disappear gradually